<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="idoshoshani-lt" name="TestAttempt" tests="2" failures="2" timestamp="26 Nov 2017 09:21:06 GMT" time="156.549" errors="0">
  <testcase name="@AfterMethod tearDown" time="0.0" classname="com.myTests.NativeTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.experitest.client.ExecutionClient.generateReport(ExecutionClient.java:139)
	at com.myTests.NativeTest.tearDown(NativeTest.java:135)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 12 stack frames]]>
    </failure>
  </testcase> <!-- @AfterMethod tearDown -->
  <testcase name="testNativeLogin" time="75.415" classname="com.myTests.NativeTest">
    <failure type="com.experitest.client.InternalException" message="Exception caught while executing elementSendText: Element was not identified: &amp;apos;usernameTextField&amp;apos; at zone default">
      <![CDATA[com.experitest.client.InternalException: Exception caught while executing elementSendText: Element was not identified: 'usernameTextField' at zone default
	at com.experitest.client.JavaClientInternals.executeInternally(JavaClientInternals.java:249)
	at com.experitest.client.Client.execute(Client.java:247)
	at com.experitest.client.ExecutionClient.execute(ExecutionClient.java:184)
	at com.experitest.client.Client.elementSendText(Client.java:807)
	at com.myTests.NativeTest.testNativeLogin(NativeTest.java:80)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- testNativeLogin -->
  <testcase name="OrderTest" time="6.984" classname="com.myTests.NativeTest">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.experitest.client.Client.hybridRunJavascript(Client.java:1376)
	at com.myTests.NativeTest.OrderTest(NativeTest.java:110)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- OrderTest -->
</testsuite> <!-- TestAttempt -->
